BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
```

Comment:
A grant number, or funding field seems to be missing here?

Reply:
Definitely need a field for Award number, but it would be optional - example case is at Allen Institute we also acknowledge Allen Institute for Brain Science for support, but have no specific award ids